tests/acceptance/image-sets-test.js

```diff
@@ -32,3 +33,35 @@ test('visiting /image-sets', function() {
     equal(currentURL(), '/image-set/24');
   });
 });
+
+test('delete /image-set/:id', function() {
+  expect(5);
+
+  stubRequest('delete', '/imageSets/24', function(request){
+    ok(true, 'delete api was called');
+    return this.noContent(204);
```

Comment:
Ah I see. Thanks for the info. This makes much more sense! I'll go ahead and merge then.